### PR TITLE
Improve measurement estimates in models

### PR DESCRIPTION
**Pull request by kieranklaassen**
# todo

- [ ] Check on mobile phone

### CODE REVIEW

**Comment by joostdecock** [comment]
I've added a WIP label. That way, when you're ready to merge, you can just remove it. Save you from having to update the title :)

File: cypress/integration/models.spec.js [hunk 1]
```diff
@@ -26,44 +23,103 @@ describe('Model flow', function() {
...
-      cy.get('[data-test=save]').click()
-      cy.wait('@update').then( xhr => {
-        expect(xhr.response.body.model.measurements.ankleCircumference).to.equal(240)
-        cy.get('[data-test=remove]').click({force: true})
-        cy.get('h1').should('contain', i18n['app.models'])
-      })
+      cy.get('[data-test=measurement] input').type('24{enter}')
+      cy.get('h1').should('contain', 'This is model with breasts')
+
+      // cy.route('PUT', `/models/${handle}`).as('update')
```
Comment (kieranklaassen) [inline cypress/integration/models.spec.js]: Finish

File: cypress/integration/models.spec.js [hunk 2]
```diff
@@ -26,44 +23,103 @@ describe('Model flow', function() {
...
+      //   expect(xhr.response.body.model.measurements.ankleCircumference).to.equal(240)
+      //   cy.get('[data-test=remove]').click({ force: true })
+      //   cy.get('h1').should('contain', i18n['app.models'])
+      // })
+      // cy.get('h1').should('contain', 'This is a test model')
     })
   })
 
-
+  context('when having the neck circomference set', () => {})
```
Comment (kieranklaassen) [inline cypress/integration/models.spec.js]: finish

File: src/components/models/edit-measurement.js [hunk 3]
```diff
@@ -8,26 +8,33 @@ import Button from '@material-ui/core/Button'
...
 import { useStaticQuery, graphql } from 'gatsby'
-import { MDXRenderer } from "gatsby-plugin-mdx"
+import { MDXRenderer } from 'gatsby-plugin-mdx'
 import { MDXProvider } from '@mdx-js/react'
 import MeasurementsImages from '../measurements/images'
+import neckstimate from '@freesewing/utils/neckstimate'
+import measurementDiffers from '@freesewing/utils/measurementDiffers'
 
 const EditMeasurement = props => {
+  console.log(props)
```
Comment (kieranklaassen) [inline src/components/models/edit-measurement.js]: remove

File: src/components/models/edit-measurement.js [hunk 4]
```diff
@@ -8,26 +8,33 @@ import Button from '@material-ui/core/Button'
...
+  const currentModel = props.app.models[props.model]
+  const currentMeasurement = props.measurement
+  console.log(currentModel)
```
Comment (kieranklaassen) [inline src/components/models/edit-measurement.js]: remove

File: src/components/models/edit-measurement.js [hunk 5]
```diff
@@ -49,59 +56,100 @@ const EditMeasurement = props => {
...
+        value * 10,
+        currentModel.breasts
+      ) <= 2
+
+    // Only show measurementEstimate for non measurements.neckCircumference
+    helperText = () => {
+      if (currentMeasurement != 'measurements.neckCircumference') {
+        return (
+          <FormattedMessage
+            id="app.weEstimate"
```
Comment (kieranklaassen) [inline src/components/models/edit-measurement.js]: @joostdecock Is this sufficient or do I need to add it anywhere else?

File: src/components/models/show.js [hunk 6]
```diff
@@ -264,57 +294,72 @@ const ShowModel = ({ app, model }) => {
...
-                    >
-                      <AddIcon fontSize="inherit" style={{ marginRight: '0.5rem' }} />
-                      <FormattedMessage id="measurements.neckCircumference" />
-                    </Button>
-                  </p>
-                </Blockquote>
-              ) : null}
-            </td>
+            <td></td>
+            <td>Estimate</td>
```
Comment (kieranklaassen) [inline src/components/models/show.js]: I18n

File: src/components/session/login/index.js [hunk 7]
```diff
@@ -18,7 +18,7 @@ const Login = ({ app, location }) => {
 
   const handlePasswordReset = evt => {
     evt.preventDefault()
-    app.backend.recoverAccount(evt.target[1].value)
+    app.backend.recoverAccount(evt.target[0].value)
```
Comment (kieranklaassen) [inline src/components/session/login/index.js]: Not sure what this is, check

File: src/components/models/show.js [hunk 8]
```diff
@@ -179,28 +210,29 @@ const ShowModel = ({ app, model }) => {
...
+      </Button>
+
+      <h5 style={styles.heading} data-test="settings-title">
+        <FormattedMessage id="app.settings" />
+      </h5>
       <table style={styles.table} className="font-title">
         <tbody>
-          <tr>
-            <td style={{ width: '225px' }}>&nbsp;</td>
-            <td style={{ width: '100%' }}>&nbsp;</td>
```
Comment (kieranklaassen) [inline src/components/models/show.js]: From @joostdecock :

> On the edit model page, can we apply a max-width to all columns but the measurement name

File: src/components/models/edit-measurement.js (hunk 5 above)
Comment (joostdecock) [reply src/components/models/edit-measurement.js]: I think that's fine, though we might throw in a link there to documentation on how we do this (which remains to be written)

File: src/components/models/show.js (hunk 6 above)
Comment (joostdecock) [reply src/components/models/show.js]: I'll take care of it

**Pull request merged**